.codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: 50..100
+
+  status:
+    project:
+      default:
+        target: 90%
+        threshold: 1%
```

Comment:
Is there any reason for setting these thresholds if we're not actually going to post a status?

Reply:
Yeah, I fixed the reporting on changes and patch coverage.